schema/base/bathymetry.yaml

```diff
@@ -0,0 +1,23 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: bathymetry
+description: topographic representation of the ocean floor
+type: object
+properties:
+  id:
+    "$ref": ../defs.yaml#/$defs/propertyDefinitions/id
+  geometry:
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/cartographyContainer
```

Comment:
Just to be clear:

1. [ ] Do we expect to have a `level` property in a bathymetry feature?
2. [ ] What known `cartography` sub-properties will be included on a bathymetry feature?

Reply:
- im not sure level applies here since we have depth, unless they'd all be `0` ??
- none that im aware of